apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/java/gov/cms/bfd/pipeline/rda/grpc/sink/direct/AbstractClaimRdaSinkTest.java

```diff
@@ -0,0 +1,253 @@
+package gov.cms.bfd.pipeline.rda.grpc.sink.direct;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import com.codahale.metrics.Meter;
+import com.codahale.metrics.MetricRegistry;
+import gov.cms.bfd.model.rda.MessageError;
+import gov.cms.bfd.model.rda.RdaApiClaimMessageMetaData;
+import gov.cms.bfd.model.rda.RdaApiProgress;
+import gov.cms.bfd.pipeline.rda.grpc.RdaChange;
+import gov.cms.bfd.pipeline.rda.grpc.source.DataTransformer;
+import gov.cms.bfd.pipeline.sharedutils.PipelineApplicationState;
+import java.io.IOException;
+import java.time.Clock;
+import java.time.Instant;
+import java.util.List;
+import java.util.stream.Collectors;
+import javax.annotation.Nonnull;
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import org.junit.jupiter.api.Test;
+
+public class AbstractClaimRdaSinkTest {
+
+  /**
+   * Checks that claims are processed successfully when no exceptions were thrown by the {@link
+   * AbstractClaimRdaSink#transformMessage(String, Object)} invocation, and that no {@link
+   * AbstractClaimRdaSink#writeError(String, Object, DataTransformer.TransformationException)}
+   * method was not invoked.
+   */
+  @Test
+  void shouldTransformValidClaimsSuccessfully() throws IOException {
+    MetricRegistry mockMetricRegistry = mock(MetricRegistry.class);
+    Meter mockMeter = mock(Meter.class);
+    EntityManagerFactory mockEntityManagerFactory = mock(EntityManagerFactory.class);
+    EntityManager mockEntityManager = mock(EntityManager.class);
+    Clock mockClock = mock(Clock.class);
+    PipelineApplicationState mockPipelineApplicationState = mock(PipelineApplicationState.class);
```

Comment:
Shouldn't these test fixtures be defined at the class level and created in a `setUp()` method?

Reply:
It's a bit of a styling choice, and a little bit of an argument of simplicity vs isolated tests.  I tend to put most of my test setup in each test unless it's a verbose amount of data, but this is relatively tiny.